File: bin/run_phase.py
#!/usr/bin/env python3
"""
后台阶段执行入口（由 runner.run_in_background 调用）
用法：python3 bin/run_phase.py <task_id> <phase>
"""

import sys
from pathlib import Path

# 把 src 加入路径
sys.path.insert(0, str(Path(__file__).parent.parent))

from core.runner import execute_phase

if __name__ == "__main__":
    if len(sys.argv) < 3:
        print("usage: run_phase.py <task_id> <phase>")
        sys.exit(1)
    execute_phase(sys.argv[1], sys.argv[2])

File: bin/run-phase.ts
#!/usr/bin/env bun
import { installAutopilotResolver } from "../src/core/autopilot-resolver";
// 在任何用户工作流被 dynamic import 前安装别名解析器
installAutopilotResolver();

import { discover } from "../src/core/registry";
import { initDb } from "../src/core/db";
import { executePhase } from "../src/core/runner";

const [taskId, phase] = process.argv.slice(2);
if (!taskId || !phase) {
  console.error("用法: bun run bin/run-phase.ts <task_id> <phase>");
  process.exit(1);
}

initDb();
await discover();
await executePhase(taskId, phase);
